ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -316,3 +336,37 @@ def test_bincount(
         weights=x[1],
         minlength=min_length,
     )
+
+
+# percentile
+@handle_test(
+    fn_tree="functional.ivy.experimental.percentile",
+    dtype_and_x=statistical_dtype_values(function="percentile"),
+    keep_dims=st.booleans(),
+    test_gradients=st.just(False),
+    test_with_out=st.just(False),
```

Comment:
Since you have included the `out` argument in your implementation of the function, it should be tested as well. So, `test_with_out` should not be hardcoded here.